conf/map/battle/misc.conf

```diff
@@ -171,3 +171,9 @@ mon_trans_disable_in_gvg: false
 // @item Bao   # 553         # 553
 // @item Bun_  # 6115        # 6115
 case_sensitive_aegisnames: true
+
+// Hercules Channel System - @channel setopt MessageDelay <delay>
+// Sets the maximum amount of message delay (in seconds) allowed for a channel.
+// Default: 10
+// Max: 255
+channel_opt_msg_delay: 10
```

Comment:
why not put this setting into channels.conf?

Reply:
good point. I had a big derp trying to figure out where the conf for HCS is.